### DISCUSSION TITLE
Enabling full OpenGL shader-based rendering for the edit mode as well as the live game mode - is it possible?

### DISCUSSION THREAD

**Discussion by benjamind2008** [revvv/gdash-export-CrLi#64] [Ideas]
One thing I've been often wondering lately is whether it is possible to display the graphics in the editor using the same shader-based rendering that you see in the live game, as a feature that could be turned on (OpenGL rendering as seen in the live game) or off (standard software based as it is now when you're in the editor).

This would be wonderful if it were possible. One could see exactly how the game would appear whilst it was being edited. The settings I use are just a standard phosphor rendering and it is beautiful to look at and reminds me of the old tube TVs I used to play bounderdash on when it came out all those years ago.

Any word on this? It seems like a good idea. Is it possible?

**Comment by revvv** [comment]
The idea is great. Unfortunately I have no time to implement it. I keep the discussion open for now. (Sorry for my late answer, completely forgotten to respond.)

**Reply by benjamind2008** [reply]
I knew something was holding up development. The project hasn't been updated since October last year, I knew it had stopped as I came here afterwards and saw no new updates.

The program runs beautifully though and it's a bonus if this can be done as well, and if someone did have the time to bring this feature into the program, it would be well received. It's more stable than the old 2013 release that I had been using before, so it's come a long way.
